### DISCUSSION TITLE
Allure Report Customization

### DISCUSSION THREAD

**Discussion by yoadimpen** [allure-framework/allure2#1418] [Questions & Support]
Hi! I've been working on an "extension" that I mentioned in issue #1142.

Currently I am expanding an Allure report with a new widget in the Overview tab and a Coverage tab.
The widget is shown in the first screenshot as well as the coverage tab but if I try to access that tab a white page loads, without even the navigation panel. I want to include a widgets grid view to this new tab and add some widgets to it but I don't know if that is even possible.

I have seen the documentation and many examples adding a tab with a tree layout but none with the widgets grid, so I am starting to think that might not be possible. I have seen how Allure generates those tabs and I have tried to replicate and adapt it to my situation but nothing has worked so far.

![allure screenshot](https://user-images.githubusercontent.com/56016797/112006129-b911cd80-8b23-11eb-92b1-2b1c76528867.png)

Is it feasible adding a custom tab with a widget grid view? Thanks in advance.

**Comment by MalikMlitat** [comment]
@yoadimpen did you publish your plugin? I am searching for a way to show tests coverage within allure report, thanx
